src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE books (
+id INT NOT NULL AUTO_INCREMENT,
```

Comment:
```suggestion
id INT AUTO_INCREMENT,
```
It is primaryKey, so NOT NULL redundant

Reply:
ok thx)